ImageConverter/ImageConverter/Image.cs

```diff
@@ -1,13 +1,16 @@
-﻿namespace ImageConverter;
+﻿using System.Reflection.Metadata.Ecma335;
+
+namespace ImageConverter;
 
 public class Image
 {
-    public Pixel[,] Pixelmap { get; }
+    private Pixel[,] pixelmap;
     public int ColorMaxValue { get => 255; }
-    public int Hieght => Pixelmap.GetLength(0);
-    public int Width => Pixelmap.GetLength(1);
+    public int Hieght => pixelmap.GetLength(0);
+    public int Width => pixelmap.GetLength(1);
+    public Pixel this[int i,int j] => pixelmap[i,j]; 
     public Image(Pixel[,] pixelmap)
```

Comment:
Check whether image pixel map changes after changing passed pixel map after image creation

Reply:
I checked and its works the way you write when after creating image I change pixel map which I pass to image it changes image pixel map too. Solve in n9c1b6a18.